docs/current/api/990050-index.md

```diff
@@ -0,0 +1,30 @@
+---
+slug: /api
+---
+
+# Dagger GraphQL API
+
+## What is the Dagger GraphQL API?
+
+Dagger exposes its functionality through a GraphQL API. GraphQL serves as a low-level language-agnostic API framework for all Dagger operations.
+
+In practice, interacting with the API directly is optional. You can instead use a Dagger SDK for one of the supported languages, or use the Dagger CLI.
+
```

Comment:
A simple stack diagram here would be useful.

Reply:
https://github.com/mermaid-js/mermaid/issues/3307  :eyes: